API to get tenants credentials from env variables (#49)
oat-sa/extension-tao-deliver-connect: README.md

To be independent workers has be to launched as a separate processes: 
- `sudo -u www-data php index.php 'oat\taoTaskQueue\scripts\tools\RunWorker' --queue=ng_publication` 
- `sudo -u www-data php index.php 'oat\taoTaskQueue\scripts\tools\RunWorker' --queue=queue`

#### TanantAPI configuration

> There are 3 implementations exists to work with a Tenant API,  configurable at `config/taoDeliverConnect/DeliverTenantService.conf.php`

Reviewer: ```suggestion
There are 3 existing implementations that work with the Tenant API configurable at `config/taoDeliverConnect/DeliverTenantService.conf.php`
```

> 1. `\oat\taoDeliverConnect\model\api\TenantHttpAPI` used to retrieve data based on agreed Rest contract

Reviewer: ```suggestion
1. `\oat\taoDeliverConnect\model\api\TenantHttpAPI` is used to retrieve data based on agreed Rest contract
```

        
              
2. `\oat\taoDeliverConnect\model\api\TenantConfigurableAPI` uses plain configuration in tao native format 
3. `\oat\taoDeliverConnect\model\api\TenantDriverApiReader` access with a configurable KV driver to storage, f.e. to the environment variables  
        
       return new oat\taoDeliverConnect\model\DeliverTenantService(
            [
                'tenant_api' => new TenantDriverApiReader(
                    [
                        TenantDriverApiReader::OPTION_DRIVER => EnvironmentVariableKVDriver::class
                    ]
                ),
            ]
        );
        
   Sample data in `DotEnv` format:
   
   
        DELIVERTENANT_COUNT=2
         
        DELIVERTENANT_0_ID=1
        DELIVERTENANT_0_LABEL=coool_label
        DELIVERTENANT_0_CUSTOMER_ID=77
        DELIVERTENANT_0_ROOT_URL='http://deliver.docker.localhost/api/v1/'
         

>         DELIVERTENANT_0_LTI_CREDENTIALS_COUNT=2

Reviewer: maybe we can add some comments that will ease up the work for the next developer
